src/js/components/Select/index.d.ts

```diff
@@ -8,14 +8,27 @@ import {
   PlaceHolderType,
 } from '../../utils';
 
+interface onChangeEvent extends React.ChangeEvent<HTMLSelectElement> {
+  value: any;
+  option: any;
```

Comment:
Can we please add comments on why this is using `any`?

Reply:
Added in the PR description